village/src/App.js

```diff
@@ -11,6 +11,19 @@ class App extends Component {
       smurfs: [],
     };
   }
+
+  componentDidMount() {
+    axios
+        .get('http://localhost:3333/smurfs')
+        .then(response => {
+            this.setState(() => ({ smurfs: response.data }));
```

Comment:
You don't really need to use an anonymous function within setState in this case, so for readability try to keep your code as minimalist as you can.